test/dlc_tests/benchmark/bai/tensorflow/training/test_performance_tensorflow_training.py

```diff
@@ -0,0 +1,12 @@
+import re
+
+from invoke.context import Context
+from test.test_utils.benchmark import single_node_benchmark_executor
+
+
```

Comment:
will need to add a marker to run only on nightly.

Check out test_utils/__init__.py's is_mainline_context(), for instance. You can add a new condition called is_nightly_context. Then, you would add a skip marker like so.

```
@pytest.mark.skipif(not is_nightly_context(), reason='Benchmarks will only run nightly')
```